services/submission.js

```diff
@@ -2072,6 +2062,33 @@ class Collaborators {
     }
 }
 
+// TODO remove temporary for QA
```

Comment:
Do there need to be removed before merging?

Reply:
QA testing purpose. We will remove it.